.gitignore

```diff
@@ -2,4 +2,7 @@ node_modules
 
 # environment variables
 .env
-.env.production
\ No newline at end of file
+.env.production
+
+dist
+/dist
```

Comment:
ah sorry didn' realise you did this also. you can a conflict now. just undo this change and it should be good. let me know if you need help.

Reply:
yer i saw you did it and wants sure of it would conflict or not